plugins/aggregators/derivative/derivative_test.go

```diff
@@ -365,10 +367,11 @@ func TestAddMetricsResetsRollOver(t *testing.T) {
 
 func TestCalculatesCorrectDerivativeOnTwoConsecutivePeriods(t *testing.T) {
 	acc := testutil.Accumulator{}
-	period, _ := time.ParseDuration("10s")
+	period, err := time.ParseDuration("10s")
+	require.NoError(t, err)
 	derivative := NewDerivative()
 	derivative.Log = testutil.Logger{}
-	err := derivative.Init()
+	err = derivative.Init()
 	require.NoError(t, err)
```

Comment:
As a note, this is even better I think
```suggestion
	require.NoError(t, derivative.Init())
```
Should not necessarily be done in this PR more as a hint for the future. :-)